docs/resources/cloud_provider_aws_cloudwatch_scrape_job.md

```diff
@@ -139,11 +140,13 @@ resource "grafana_cloud_provider_aws_cloudwatch_scrape_job" "test" {
 
 - `custom_namespace` (Block List) Zero or more configuration blocks to configure custom namespaces for the CloudWatch Scrape Job to scrape. Each block must have a distinct `name` attribute. When accessing this as an attribute reference, it is a list of objects. (see [below for nested schema](#nestedblock--custom_namespace))
 - `enabled` (Boolean) Whether the CloudWatch Scrape Job is enabled or not.
+- `export_tags` (Boolean) When enabled, AWS resource tags are exported as Prometheus labels to metrics formatted as `aws_<service_name>_info`.
 - `service` (Block List) One or more configuration blocks to configure AWS services for the CloudWatch Scrape Job to scrape. Each block must have a distinct `name` attribute. When accessing this as an attribute reference, it is a list of objects. (see [below for nested schema](#nestedblock--service))
 
 ### Read-Only
 
-- `id` (String) The Terraform Resource ID. This has the format "{{ stack_id }}:{{ job_name }}".
+- `disabled_reason` (String) When the CloudWatch Scrape Job is disabled, this will show the reason that it is in that state.
```

Comment:
ah so it's a read only field?

Reply:
Yeah exactly, in the resource, which is the Terraform structure type that allows a user to CRUD a thing with Terraform as a client, this is exported a a read-only field returned back from the API; its value can't be specified by a user. For the data source, which is a read-only thing inherently (intended for cases where customers want to look up fields from existing jobs not necessarily managed by Terraform), it's also a read-only field.